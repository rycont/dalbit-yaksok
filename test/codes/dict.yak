객체 = {
    이름: '홍길동'
    나이: 30
    주소: '서울시 강남구'
}
    
객체['이름'] 보여주기
객체['나이'] 보여주기
객체['주소'] 보여주기

객체['이름'] += "이였던것"
객체['이름'] 보여주기

객체['나이'] += 1
객체['나이'] 보여주기

새_객체 = {
    이름: '김철수'
    나이: 객체['나이'] + 20,
    주소: "부산시 해운대구",
    자식새끼: [{
        이름: '김영희',
        나이: 5,
        주소: "부산시 해운대구",
    }, {
        이름: '김철수',
        나이: 7,
        주소: "부산시 해운대구"
        자격: ['수영', '축구']
    }]
}

새_객체['이름'] 보여주기
새_객체['나이'] 보여주기
새_객체['주소'] 보여주기

우리_집 = [{
    이름: "광솽관",
    나이: 3,
    주소: "서울시 강남구",
}, {
    이름: '닝구',
    나이: 2,
    주소: "서울시 강남구",
    자격: ['수영', '축구']
}, {
    이름: '우리집',
}]

우리_집[1]['이름'] 보여주기