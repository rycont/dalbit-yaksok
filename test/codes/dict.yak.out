홍길동
30
서울시 강남구
홍길동이였던것
31
김철수
51
부산시 해운대구
닝구
